src/Plugin.php

```diff
@@ -63,6 +63,16 @@ public static function wp_enqueue_scripts() {
     wp_enqueue_script('gallerya-script-custom', static::getBaseUrl() . '/dist/scripts/script.min.js', ['gallerya-script-libs'], FALSE, TRUE);
   }
 
+  public static function post_gallery_init() {
+    $url = home_url(add_query_arg(array()));
+    $id = url_to_postid($url);
```

Comment:
I know this is not the best idea in order to performance but I don't know how can I retrieve the current post type without run so expensive operation. Suggestion about? 🤔